### DISCUSSION TITLE
Database migration with Alembic

### DISCUSSION THREAD

**Discussion by abbasihamed** [fastapi/fastapi#5153] [Questions]
I want that when I add a column to one of my database tables, during the database migration with Alembic, that column will be added without deleting the database.
But after the migration, my database is completely deleted and created from scratch, and this causes the loss of my previous data

**Comment by EssaAlshammri** [comment]
make sure you're not using 
https://docs.sqlalchemy.org/en/14/core/metadata.html#sqlalchemy.schema.MetaData.drop_all

**Reply by EssaAlshammri** [reply]
```python
Base.metadata.drop_all(bind=engine)
```

I meant this.
